questions.md

```diff
@@ -0,0 +1,5 @@
+# Questions on pull requests
+- What is the definition of the base branch
+- What is the definition of the head branch
+- How do I review the pull request and see changes
+- How do I ask for changes to the submitted pull request?
```

Comment:
You can approve the PR, just give feedback, or request changes. Then the contributor edits the branch they PR'd, pushes it, resolves the comments and is good to go.